packages/fluent-theme/src/private/FluentThemeProvider.tsx

```diff
@@ -14,7 +15,9 @@ const sendBoxMiddleware = [() => () => () => SendBox];
 const FluentThemeProvider = ({ children }: Props) => (
   <WebChatTheme>
     <TelephoneKeypadProvider>
-      <ThemeProvider sendBoxMiddleware={sendBoxMiddleware}>{children}</ThemeProvider>
+      <ThemeProvider sendBoxMiddleware={sendBoxMiddleware}>
+        <WebChatDecorator>{children}</WebChatDecorator>
```

Comment:
Seems unused/not implemented right now.

How should we use `<WebChatDecorator>`? Or why `<WebChatTheme>` is not enough?

Reply:
We can reintegrate decorator to the theme, but why should we? May the decorator be useful without the theme? I thought so, but we're not going to ship the decorator yet, so for me it's the way to eliminate unused code: don't import the decorator and nothing from the workstream is included into the release (for Fluent)